File: Java_Programs/Voting.java
import java.util.Scanner;
public class Voting {
	public static void main(String[] args) {
      Scanner sc = new Scanner(System.in);
      int votingage = 18;
      System.out.print("Welcome to the voting camp \n");
      System.out.print("Enter your age: ");
       int age = sc.nextInt();
       if (age < votingage) {
    	   System.out.print("Sorry, You are not eligible for voting \n");
       } else {
    	   System.out.print("You are eligible for Voting \n");
       }
       System.out.print("Choose your Option \n");
       System.out.print("1.TDP 2.BRS \n");
       System.out.print("Enter your Option \n");
       int vote = sc.nextInt();
       switch(vote) {
       case 1: 
    	    System.out.print("1.TDP \n");
    	    break; 
    	case 2: 
            System.out.print("2.BRS \n");
            break;
       }
       System.out.print("Thanks for the Voting");
	}

}

File: Python programs/Voting.py
voter_age = 18
print("Welcome Come For the Voting Station")
age = int(input("Enter your age: "))
if age < voter_age:
                    print("Sorry, You are not eligible for voting")
else:
        print("You are eligible Voting")
print("_______________Choose your opinion_______________")
print("1.TDP")
print("2.YSRCP")
print("3.JSP")
print("4.BRS")
print("5.BJP")
print("6.Congress")
vote = int(input("Enter your option: "))
if vote == 1:
        print("You are voted for TDP")
elif vote == 2:
         print("You are voted for YSRCP")
elif vote == 3:
        print("You are voted for JSP")
elif vote == 4:
        print("You are voted for BRS")
elif vote == 5:
        print("You are voted for BJP")
elif vote == 6:
        print("You are voted for Congress")
else:
        print("_______________Invalid Option_______________")
print("Tnanks, for voting") 
